src/source-nodes.js

```diff
@@ -100,11 +100,14 @@ module.exports = (
     };
 
     let outboundReferences = note.outboundReferences;
-    brainNoteNode.outboundReferences = outboundReferences
-      // Use the slug for easier use in queries
-      .map((match) => nameToSlugMap[match.text.toLowerCase()])
-      // Filter duplicates
-      .filter((a, b) => outboundReferences.indexOf(a) === b);
+
+    // Use the slug for easier use in queries
+    let outboundReferenceSlugs = outboundReferences.map(
+      (match) => nameToSlugMap[match.text.toLowerCase()]
+    );
+
+    // Remove duplicates
+    brainNoteNode.outboundReferences = [...new Set(outboundReferenceSlugs)];
```

Comment:
Can update this to deduplicate like inbound references below if you'd like it to be consistent, although I think this is slightly more performant.

Reply:
Yeah, this seems like a good approach assuming this does dedupe it properly